auth/setup.go

```diff
@@ -229,18 +229,34 @@ func AddAdminUser(authService Service, user *model.User) (*model.Credential, err
 		return nil, fmt.Errorf("add user to group - %w", err)
 	}
 
-	// Generate and return a key pair
-	creds, err := authService.CreateCredentials(user.Username)
-	if err != nil {
-		return nil, fmt.Errorf("create credentials for %s %w", user.Username, err)
+	var creds *model.Credential
+	if user.AccessKeyID == "" {
+		// Generate and return a key pair
+		creds, err = authService.CreateCredentials(user.Username)
+		if err != nil {
+			return nil, fmt.Errorf("create credentials for %s %w", user.Username, err)
```

Comment:
Here and elsewhere, go format standard for errors is:

```suggestion
			return nil, fmt.Errorf("create credentials for %s: %w", user.Username, err)
```

Reply:
1. Done!
2. This continues the previous format.
3. So I didn't fix the rest of the file. :shrug: